tests/user_commands/tests.py

```diff
@@ -254,3 +254,12 @@ def test_no_existent_external_program(self):
         msg = 'Error executing a_42_command_that_doesnt_exist_42'
         with self.assertRaisesMessage(CommandError, msg):
             popen_wrapper(['a_42_command_that_doesnt_exist_42'])
+
+class CommandParserKwargsTests(SimpleTestCase):
+    """
+    Tests that a CommandParser instance is created with the passed kwargs.
+    """
+    def test_commandparser_is_initialized_with_passed_kwargs(self):
+        command = BaseCommand()
+        parser = command.create_parser('', 'some_command', epilog='This should be passed to CommandParser instance.')
```

Comment:
I'd extract the `epilog` string into a variable and re-use it to get the line length down.